src/libstd/time/mod.rs

```diff
@@ -95,8 +94,9 @@ impl Instant {
     ///
     /// # Panics
     ///
-    /// This function may panic if the current time is earlier than this instant
-    /// which can happen if an `Instant` is produced synthetically.
+    /// This function may panic if the current time is earlier than this
```

Comment:
What do you mean? `"    /// This function may panic if the current time is earlier than this instant,"` is 81 characters only.